docs/scout.txt

```diff
@@ -0,0 +1,199 @@
+.. _laravel-scout:
+
+===========================
+Full-Text Search with Scout
+===========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, odm, code example, text search, atlas
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the Laravel Scout feature in
+your {+odm-long+} application. Scout allows you to implement full-text
+search on your Eloquent models. To learn more, see `Laravel Scout
+<https://laravel.com/docs/{+laravel-docs-version+}/scout>`__ in the
+Laravel documentation.
+
+The Scout integration for {+odm-long+} provides the following
+functionality:
+
+- Provides an abstraction to create search indexes on documents in
+  MongoDB collections and external search engines. In MongoDB, this feature
+  allows you to create :atlas:`Atlas Search indexes
+  </atlas-search/manage-indexes/>`.
+
+- Allows you to automatically replicate data from MongoDB into a
+  search engine such as `Meilisearch <https://www.meilisearch.com/>`__
+  or `Algolia <https://www.algolia.com/>`__. You can use a MongoDB Eloquent
+  model as the source to import and index.
+
+.. note:: Deployment Compatibility
+
+   You can use Laravel Scout only when you connect to MongoDB Atlas
+   clusters. This feature is not available for self-managed or
+   serverless deployments.
+
+Scout for Atlas Search Tutorial
+-------------------------------
+
+This section demonstrates how to use the Scout integration in your
+application to support Atlas Search queries.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install Scout package
```

Comment:
S: to make this step a [full sentence](https://www.mongodb.com/docs/meta/style-guide/style/tasks/#steps)
```suggestion
   .. step:: Install the Scout package
```